tardis/plasma/properties/atomic.py

```diff
@@ -108,8 +109,17 @@ def calculate(self, atomic_data, selected_atoms):
 class IonizationData(ProcessingPlasmaProperty):
     name = 'ionization_data'
 
-    def calculate(self, atomic_data, selected_atoms):
+    def calculate(self, atomic_data, levels):
         if self.value is not None:
             return self.value
         else:
             return atomic_data.ionization_data
+
+class ZetaData(ProcessingPlasmaProperty):
+    name = 'zeta_data'
```

Comment:
I think they shouldn't be there, but we are in the process of changing the dataset anyways - so for now I would leave that. I think the `ZetaData` stuff is more sensible now.